src/vse_sync_pp/analyzers/ppsdpll.py

```diff
@@ -2,29 +2,16 @@
 
 """Analyze ppsdpll log messages"""
 
-from .analyzer import Analyzer
+from .analyzer import TimeErrorAnalyzer
 
-class PhaseOffsetTimeErrorAnalyzer(Analyzer):
-    """Analyze DPLL phase offset time error"""
-    id_ = 'ppsdpll/phase-offset-time-error'
-    parser = 'dpll/phase-offset'
+class PpsDpllAnalyzer(TimeErrorAnalyzer):
+    """Analyze DPLL time error"""
+    id_ = 'ppsdpll/time-error'
+    parser = 'dpll/time-error'
     def __init__(self, config):
         super().__init__(config)
-        # required system time output accuracy
-        accuracy = config.requirement('time-error-in-locked-mode/ns')
-        # exclusive upper bound on absolute time error for any sample
-        self._unacceptable = accuracy
+        self._lockid = frozenset({3})
     def prepare(self, rows):
```

Comment:
The base class uses config parameter for the transient window, which this prepare method ignores. Either this class should specify exactly and only the config it needs, or it should use all of the config the base class takes. I think this should be the latter, the prepare method should be in the base and should discard all samples in the transient period.

Reply:
Fixed.